### PR TITLE
Add input form for editting display name in Slack channel

### PR DESCRIPTION
**Pull request by dchiji**
Ref. #35 

Enable to edit display name for each slack notifications.
Please review it.

### CODE REVIEW

File: schema/tokite_rules.schema [hunk 1]
```diff
@@ -5,6 +5,7 @@ create_table "tokite_rules", force: :cascade do |t|
   t.string   "channel",         limit: 100,  null: false
   t.string   "icon_emoji",      limit: 20,   null: false, default: ""
   t.string   "additional_text", limit: 200,  null: false, default: ""
+  t.string   "display_name",    limit: 60,   null: false, default: ""
```
Comment (eagletmt) [inline schema/tokite_rules.schema]: display_name must be added to the last.
https://github.com/cookpad/tokite/pull/41#discussion_r195951523

Comment (dchiji) [reply schema/tokite_rules.schema]: Thanks, I forgot it.

File: app/models/tokite/hook.rb [hunk 2]
```diff
@@ -32,6 +32,7 @@ def fire!
         if payloads.none? {|payload| payload[:channel] == rule.channel && payload[:emoji] == emoji && payload[:additional_text] == additional_text }
           payloads << {
             channel: rule.channel,
+            username: (rule.display_name == "") ? "tokite" : rule.display_name,
```
Comment (hogelog) [inline app/models/tokite/hook.rb]: (Either way is OK) You can write `rule.display_name.presence || "tokite"` here.
http://api.rubyonrails.org/classes/Object.html#method-i-presence

File: app/models/tokite/hook.rb [hunk 3]
```diff
@@ -40,8 +41,20 @@ def fire!
...
+          attachments: payload[:attachments],
+          username: payload[:username]
+        )
+        notify!(
+          channel: payload[:channel],
+          text: payload[:additional_text],
+          icon_emoji: payload[:emoji],
+          parse: "full",
+          username: payload[:username]
+        ) if payload[:additional_text].present?
```
Comment (hogelog) [inline app/models/tokite/hook.rb]: I feel bit strange use suffix if when multiline exp

**Pull request merged**